lib/liquid/expression.rb

```diff
@@ -8,6 +8,8 @@ class Expression
       'empty'.freeze => :empty?
     }
 
+    INVERTED_LITERALS = LITERALS.invert
```

Comment:
Having `""` as the inverse of `nil` makes sense to me - would be nice to preserve what they used originally though. Will think about that a bit.

Fortunately hash enumeration order is defined as the insertion order in 1.9+, so the inverse mapping is deterministic at least.